x-pack/plugin/esql/qa/testFixtures/src/main/resources/union_types.csv-spec

```diff
@@ -1397,6 +1397,23 @@ event_duration:long | _index:keyword       | ts:date                   | ts_str:
 8268153             | sample_data_ts_nanos | 2023-10-23T13:52:55.015Z  | 2023-10-23T13:52:55.015123456Z  | 1698069175015123456 |  172.21.3.15   |  172.21.3.15
 ;
 
```

Comment:
You might want to look at inlinestats.csv-spec file and search for `https://github.com/elastic/elasticsearch/issues/133973`. There are some ignored tests in there that need a check.